[2023-07-11 00:25:34,121] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 00:25:34] "GET /aplicacoes?page=undefined&per_page=undefined HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 00:27:31,634] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 00:27:31] "POST /aplicacao_filtros HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 00:27:34,170] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 00:27:34] "POST /aplicacao_filtros HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:03:27,439] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:03:27] "POST /aplicacao HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:04:16,454] ERROR log_exception() L1449 Exception on /aplicacao [PUT] - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py L1449
Traceback (most recent call last):
  File "/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask_openapi3/scaffold.py", line 273, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask_openapi3/do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/app.py", line 295, in upd_aplicacao
    query = request.json_data
            ^^^^^^^^^^^^^^^^^
AttributeError: 'Request' object has no attribute 'json_data'
[2023-07-11 20:04:16,466] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:04:16] "[35m[1mPUT /aplicacao?id=1&nome=SAP%20SuccessFactors&sigla=SSFS&descricao=SAP%20Success%20Factor&status=A HTTP/1.1[0m" 500 - - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:09:28,142] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:09:28] "GET /aplicacoes?page=undefined&per_page=undefined HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:09:37,207] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:09:37] "OPTIONS /aplicacao HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:09:37,224] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:09:37] "PUT /aplicacao HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:09:37,235] INFO _log() L224  127.0.0.1 - - [11/Jul/2023 20:09:37] "GET /aplicacoes?page=undefined&per_page=undefined HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-07-11 20:12:19,739] INFO _log() L224   * Detected change in '/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/app.py', reloading - call_trace=/home/leandro/Documentos/Treinamentos/PÃ³s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
